tests/spec/puppeteer-spec.js

```diff
@@ -147,4 +147,24 @@ describe("Puppeteer basic tests for the Python Editor.", function() {
         expect(codeName).toEqual("main");
     });
 
+    it("Saves a python file with the correct filename", async function(){
+        const page = await global.browser.newPage();
+        await page._client.send('Page.setDownloadBehavior', {behavior: 'allow', downloadPath: './spec/test-files/'});
```

Comment:
Could you create a new folder inside `test-files` and use that one?
That way we can add it to the gitignore file in case sometimes there is a rogue file left there after the tests are done (can happen if a test fails or the test under code crashes while running).

Reply:
Done.